practice.html

```diff
@@ -19,6 +19,21 @@ <h2 class="subtitle">Practice your typing skills!</h2>
         <div class="dynamic-text-box">
             <p id="quote"></p>
         </div>
+        <div id="levelSelector">
+                <h3>Choose difficulty level:</h3>
+                <div class="button-group">
+                    <input type="radio" id="levelOne" name="radio-button">
+                    <label for="levelOne" class="radio-label">1</label>
```

Comment:
Add some more relevent names to the difficulty level. 

like `Beginner`, `intermediate`, `advanced`.